lib/atoms/font/makeFontSizes.ts

```diff
@@ -1,74 +1,60 @@
-import merge from 'lodash/merge';
+import { TextDefinition } from './../utils/alignTextToGrid';
 import alignTextToGrid from '../utils/alignTextToGrid';
-import createResponsiveRules from '../utils/createResponsiveRules';
 import { px } from '../utils/toUnit';
-import each from 'lodash/each';
-import { Tokens } from 'lib/themes/theme';
+import makeDesktopRules from '../utils/makeDesktopRules';
+import { Tokens } from '../../themes/theme';
 
-export default (tokens: Tokens) => {
-  const rules: Array<{ [index: string]: object }> = [];
-
-  each(
-    {
-      ...tokens.text,
-      ...tokens.heading
-    },
-    (type, typeName) => {
-      const {
-        fontSize: mobileFontSize,
-        lineHeight: mobileLineHeight
-      } = alignTextToGrid(type.mobile, tokens);
-      const {
-        fontSize: desktopFontSize,
-        lineHeight: desktopLineHeight
-      } = alignTextToGrid(type.desktop, tokens);
+const getFontSizeRules = (textDefinition: TextDefinition, tokens: Tokens) => {
```

Comment:
Should we be explicit with the return types on these functions, eg `CSSProperties`, given what we saw yesterday with the baseline stuff

Reply:
In this case, the type is inferred correctly. Would rather use inference where possible. Once we start using the CSSProperty types we may need to review a bunch of things. But for now, I think this is fine.